rules/explicit-length-check.js

```diff
@@ -62,9 +72,15 @@ const lengthSelector = [
 
 function getBooleanAncestor(node) {
 	let isNegative = false;
-	while (isLogicNotArgument(node)) {
-		isNegative = !isNegative;
-		node = node.parent;
+	while (true) {
```

Comment:
@sindresorhus How do we write this? `eslint-disable` ?

Reply:
Yes, `eslint-disable`. Because in most cases when you want to use `while (true)`, there's usually a safer alternative.